docs/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.4.4
```

Comment:
You didn't update the `setup.py` file with `0.4.4` - we'll need that to publish the change

Reply:
I will do